recipes-hosted-login/mender-hosted-login/mender-hostedlogin-0.1/menderLogin.py

```diff
@@ -0,0 +1,235 @@
+import sys
```

Comment:
Normally I would put source files like these in a separate repository, but since this is fairly specialized I guess we can keep it here for now.